Documentation/guides/building-apps/build-properties.md

```diff
@@ -1351,6 +1351,32 @@ To suppress the default AOT profiles, set the property to `false`.
 
 Added in Xamarin.Android 10.1.
 
+## AndroidUseDesignerAssembly
+
+A bool property which controls if the build system will generate an
+`_Microsoft.Android.Resource.Designer.dll` as apposed to a `Resource.Designer.cs` file. The benefits of this are smaller applications and
+faster startup time.
+
+The default value is `true`.
+
+This setting is not backward compatible with Classic Xamarin.Android.
+So as a Nuget Author it is recommended that you ship three versions of
+the assembly if you want to maintain backward compatibility.
+One for MonoAndroid one for net6.0-android and
+one for net8.0-android. You can do this by using [Xamarin.Legacy.Sdk](https://www.nuget.org/packages/Xamarin.Legacy.Sdk).
+
+```
+<TargetFrameworks>monoandroid90;net6.0-android;net8.0-android</TargetFrameworks>
+```
+
+Alternatively turn this setting off until such time as both Classic and
+net7.0-android have been deprecated.
+
+.NET Projects which choose to turn this setting off will not be able to
```

Comment:
This paragraph should explicitly mention that *App* projects *cannot* turn this property off; it is required.

I'm not sure why *Library* projects would have this requirement.  It doesn't immediately make sense to me.

Reply:
Even under .net 8 users can still turn this off if they need to. I did it that way just in case we hit some problems. 
This is true to library projects as well. Turning it off will allow the user to use the old system.

What I'm trying to say here is if a user DOES turn it off, they will NOT be able to comsume projects they do use it, but I guess we added a new Build Error for that case.... So perhaps we don't need to even mention it.